unite@hardpixel.eu/buttons.js

```diff
@@ -178,22 +178,49 @@ export const TrayIndicator = GObject.registerClass(
 
 export const WindowControls = GObject.registerClass(
   class UniteWindowControls extends PanelMenu.Button {
-    _init() {
+    _init(params) {
       super._init(0.0, null, true)
 
       this._controls = new St.BoxLayout({ style_class: 'window-controls-box' })
       this.add_child(this._controls)
 
       this.add_style_class_name('window-controls')
       this.remove_style_class_name('panel-button')
+      this._iconScaleWorkaround = params.iconScaleWorkaround
+      this._actionIcons = params.actionIcons
```

Comment:
Prefer using `setControlThemeParams` to set these since it is a workaround and might be removed in the future.